vhdbuilder/packer/packer_source.sh

```diff
@@ -160,27 +160,31 @@ copyPackerFiles() {
       KATA_INITRD_SRC=/home/packer/kata-containers-initrd-base.img
       KATA_INITRD_DEST=$KATA_CONFIG_DIR/kata-containers-initrd.img
       cpAndMode $KATA_INITRD_SRC $KATA_INITRD_DEST 0755
-
-      KATACC_IMAGE_SRC=/home/packer/kata-containers.img
-      KATACC_IMAGE_DEST=$KATACC_CONFIG_DIR/kata-containers.img
-      cpAndMode $KATACC_IMAGE_SRC $KATACC_IMAGE_DEST 0755
-    elif [[ $OS_VERSION == "3.0" ]]; then
-      KATA_IMAGE_SRC=/home/packer/kata-containers.img
-      KATA_IMAGE_DEST=$KATA_CONFIG_DIR/kata-containers.img
-      cpAndMode $KATA_IMAGE_SRC $KATA_IMAGE_DEST 0755
-
-      KATACC_IMAGE_SRC=/home/packer/kata-containers-cc.img
-      KATACC_IMAGE_DEST=$KATACC_CONFIG_DIR/kata-containers.img
-      cpAndMode $KATACC_IMAGE_SRC $KATACC_IMAGE_DEST 0755
     fi
 
+    IGVM_BIN_SRC=/home/packer/kata-containers-igvm.img
+    IGVM_BIN_DEST=$KATACC_CONFIG_DIR/kata-containers-igvm.img
+    cpAndMode $IGVM_BIN_SRC $IGVM_BIN_DEST 0755
+
     IGVM_DEBUG_BIN_SRC=/home/packer/kata-containers-igvm-debug.img
     IGVM_DEBUG_BIN_DEST=$KATACC_CONFIG_DIR/kata-containers-igvm-debug.img
     cpAndMode $IGVM_DEBUG_BIN_SRC $IGVM_DEBUG_BIN_DEST 0755
 
-    IGVM_BIN_SRC=/home/packer/kata-containers-igvm.img
-    IGVM_BIN_DEST=$KATACC_CONFIG_DIR/kata-containers-igvm.img
-    cpAndMode $IGVM_BIN_SRC $IGVM_BIN_DEST 0755
+    KATA_IMAGE_SRC=/home/packer/kata-containers.img
+    KATA_IMAGE_DEST=$KATA_CONFIG_DIR/kata-containers.img
+    cpAndMode $KATA_IMAGE_SRC $KATA_IMAGE_DEST 0755
+
+    KATA_DBG_IMAGE_SRC=/home/packer/kata-containers-debug.img
+    KATA_DBG_IMAGE_DEST=$KATA_CONFIG_DIR/kata-containers-debug.img
+    cpAndMode $KATA_DBG_IMAGE_SRC $KATA_DBG_IMAGE_DEST 0755
+
+    KATACC_IMAGE_SRC=/home/packer/kata-containers-cc.img
+    KATACC_IMAGE_DEST=$KATACC_CONFIG_DIR/kata-containers-cc.img
```

Comment:
I am not sure we want this to be kata-containers-cc.img. We should not need to be required to modify an existing naming convention with this change, should we?